src/main/java/nextstep/step2/view/ResertView.java

```diff
@@ -0,0 +1,93 @@
+package nextstep.step2.view;
+
+import java.util.List;
+import nextstep.step2.domain.GameParticipants;
+import nextstep.step2.domain.Line;
+
+public class ResertView {
+
+  private final int DEFAULT_NAME_LENGTH = 1;
+  private final String EMPTY_LINE_SYMBOLE = " ";
+  private final String LADDER_CONNECT_SYMBOLE = "-";
+  private final String LADDER_SYMBOLE = "|";
+
+  public static void print(GameParticipants gameParticipants) {
+    for (int i = 0; i < gameParticipants.getParticipantsSize(); i++) {
+      System.out.println(gameParticipants.getParticipant(i).getParticipantName());
+    }
+  }
+
+  public void printLines(GameParticipants gameParticipants, List<Line> lines) {
+
+    int ladderWidth = getlongestParticipantsNameLength(gameParticipants);
+
+    printParticipants(gameParticipants, ladderWidth);
+
+    for (int i = 0; i < gameParticipants.getParticipantsSize(); i++) {
+      printLine(lines.get(i), ladderWidth);
+    }
+  }
+
+  private int getlongestParticipantsNameLength(GameParticipants gameParticipants) {
```

Comment:
사용자 이름의 길이를 생각한 간격조절 👍 세심하고 너무 좋은데요?? 👍 
컨벤션도 잡고 가시죠 😄 
```suggestion
  private int getLongestParticipantsNameLength(GameParticipants gameParticipants) {
```